src/newgrp.c

```diff
@@ -72,7 +72,7 @@ static void usage (void)
 	if (is_newgrp) {
 		(void) fputs (_("Usage: newgrp [-] [group]\n"), stderr);
 	} else {
-		(void) fputs (_("Usage: sg group [[-c] command]\n"), stderr);
+		(void) fputs (_("Usage: sg [-] group [[-c] command]\n"), stderr);
```

Comment:
This should probably be `[-|-l]`, right?  Which could also be expressed as `[-[l]]`, although I think the former would be more readable.